docs/docs-content/clusters/cluster-management/cluster-map-filters.md

```diff
@@ -0,0 +1,155 @@
+   | **Architecture**    | Filter and display clusters based on the host architecture.                                                                                                                                             | **AMD64** and **ARM64**                                                                  |
+   | **Country**         | Filter and display clusters based on the cluster location country.                                                                                                                                      | Any country where a cluster is deployed.                                                 |
+   | **Region**          | Filter and display clusters based on the cluster location region                                                                                                                                        | Any region where a cluster is deployed                                                   |
+   | **Projects**        | Filter and display clusters based on the project they are deployed to. This filter is only available in tenant scope.                                                                                   | Any project belonging to this tenant                                                     |
+
+   Just as with the list view filters, you can combine as many filters as you require and only clusters that satisfy all
+   filter requirements are displayed.
+
+## Clusters Map View
+
+Palette provides a **Clusters Map View** as an alternative way to visualize and search your clusters. The map is
+constructed based on the configured cluster location and region. The map view provides filters that allow you to filter
+and search through your deployed clusters. You can also search your clusters by name.
+
+### Prerequisites
+
+- A [Palette](https://console.spectrocloud.com) account with the permissions to view and list clusters. Refer to the
+  [Roles and Permissions](../../user-management/palette-rbac/project-scope-roles-permissions.md) guide for more
+  information.
+- One or more active Kubernetes clusters in Palette.
+
+### Enablement
+
+1. Login to [Palette](https://console.spectrocloud.com).
+
+2. Select **Clusters** from the left **Main Menu**. Clusters deployed to data centers or edge hosts need to have their
+   locations set manually. Expand the following section to learn how to configure cluster locations. You can skip this
+   step for clusters deployed to public clouds.
+
+   <details>
+
+   <summary>Configure Cluster Location</summary>
+
+   Select the cluster to you want to update.
+
+   Click on **Settings**. Then, select **Cluster Settings**. The **Settings** pane appears.
+
+   Select the **Location** option. Search for the cluster location and click on **Save Changes**.
+
+   Repeat these steps for all the clusters that are missing a location.
```

Comment:
Could these be changed to numbered steps?

Just wondering if there was a reason if they weren't.

Reply:
Added! Thanks for the suggestion!